frontend/src/App.vue

```diff
@@ -41,6 +41,15 @@ export default {
 }
 </script>
 <style>
+.gdd-toaster-title {
+  color: #ffffff !important;
+}
+.gdd-toaster-body {
+  color: #ffffff !important;
+}
+.gdd-toaster {
+  color: #ffffff;
+}
```

Comment:
Can we move this in scss? Or is that not useful?

Reply:
I suggest to do this with the new layout